client/my-sites/plugins/plugin-settings-link/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+	displayName: 'PluginSettingsLink',
+
+	propTypes: {
+		link: React.PropTypes.string.isRequired,
+	},
+
+	render() {
+		if ( ! this.props.link ) {
```

Comment:
since this is a required prop, the only way this would fail is if we send an empty string. is that the intention for the if structure? if not, maybe we could take it out completely as React should warn us when developing if we fail to pass it.